src/Mono.Android/Mono.Android.csproj

```diff
@@ -73,6 +78,11 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' ">
+    <ProjectReference Include="..\..\external\Java.Interop\src\Java.Interop\Java.Interop.csproj" />
+    <PackageReference Include="System.Drawing.Common" Version="4.7.0" />
```

Comment:
I believe that this reference was only needed to ensure this attribute compiled successfully:

https://github.com/xamarin/xamarin-android/blob/master/src/Mono.Android/Properties/AssemblyInfo.cs.in#L29

This raises another question though, should we still be including these typeforwards, and the [`System.Drawing`](https://github.com/xamarin/xamarin-android/commit/c7a41e59f5391301008474f1469405aa5c7a5b62) sources when targeting `netstandard2.1` (or `net5` in the future)?

Reply:
I was able to remove the `System.Drawing.Common` reference when targeting `netcoreapp3.1`, however I'm still not sure if we should be including the sources/typeforwards when building against this TargetFramework.